README.md

```diff
@@ -1,6 +1,7 @@
 # JCasC Demo on CloudBees Jenkins Distribution
 
-![Docker Pulls](https://img.shields.io/docker/pulls/cloudbees/cjd-jcasc-demo?logo=cloudbees)
+[![GitHub release](https://img.shields.io/github/release/cloudbees-oss/cjd-jcasc-demo.svg?label=release)](https://github.com/cloudbees-oss/cjd-jcasc-demo/releases/latest)
```

Comment:
The image for GH releases is not properly visualized?

Reply:
You need to refresh it few times before it works. Shields.io has horrible stability with default caching settings